dbt_subprojects/dex/models/_projects/cow_protocol/arbitrum/cow_protocol_arbitrum_trades.sql

```diff
@@ -1,17 +1,17 @@
 {{  config(
-        schema = 'cow_protocol_arbitrum',
-        alias='trades',
-        materialized='incremental',
-        partition_by = ['block_month'],
-        unique_key = ['tx_hash', 'order_uid', 'evt_index'],
-        on_schema_change='sync_all_columns',
-        file_format ='delta',
-        incremental_strategy='merge',
-        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
-        post_hook='{{ expose_spells(\'["arbitrum"]\',
-                                    "project",
-                                    "cow_protocol",
-                                    \'["olgafetisova"]\') }}'
+    schema='cow_protocol_arbitrum',
+    alias='trades',
+    materialized='incremental',
+    partition_by = ['block_month'],
+    unique_key = ['tx_hash', 'order_uid', 'evt_index'],
+    on_schema_change='sync_all_columns',
+    file_format ='delta',
+    incremental_strategy='merge',
+    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "cow_protocol",
+                                \'["olgafetisova"]\') }}'
```

Comment:
Don't forget to include yourself!
```suggestion
                                \'["olgafetisova", "fleupold"]\') }}'
```

Reply:
Will do this for the ones I actually contribute to